### PR TITLE
Updated version to 0.6.1

### PR DESCRIPTION
**Pull request by scotthain**
pyasn1 0.6.1 :snowflake:

**Destination channel:** defaults

### Links

- [PKG-5702](https://anaconda.atlassian.net/browse/PKG-5702) 
- [Upstream repository](https://github.com/pyasn1/pyasn1)
- [Upstream changelog/diff](https://github.com/pyasn1/pyasn1/blob/main/CHANGES.rst)

### Explanation of changes:
- Updated version

[PKG-5702]: https://anaconda.atlassian.net/browse/PKG-5702?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bwingconda — APPROVED** [review]

**Review by Arishamays1 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: True # [py<38]
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv"
 
 requirements:
   host:
     - python
     - pip
+    - wheel
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:24]:
```suggestion
    - setuptools
    - wheel
```
Missing setuptools

File: recipe/meta.yaml [hunk 2]
```diff
@@ -31,16 +39,15 @@ test:
     - pyasn1.type
 
 about:
-  home: https://github.com/etingof/pyasn1
-  license: BSD 2-Clause
+  home: https://github.com/pyasn1/pyasn1
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:43]: FYI, the explanation of why the maintainer has been changed https://github.com/pypi/support/issues/2090. 
Changing the repo maintainer can be qualified as a security concern and should be always investigated by us.

**Review by skupr-anaconda — COMMENTED** [review]
LGTM but setuptools

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (scotthain) [reply recipe/meta.yaml:43]: Thank you! I didn't know, and I would have known where to search. I'll make sure to do that in the future.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**